src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -123,17 +126,44 @@ private static int getNumPodsForNamespace(Map<Timestamp, IntervalResults> filter
         return (int) Math.ceil(max_pods_cpu);
     }
 
-    private void init() {
-        // Add new models
+    private void DefaultLoadRecommendationModel() {
```

Comment:
Method name should always begin with a small letter and rename it to make more sense. Something like:
`loadDefaultRecommendationModels`